### ISSUE TITLE
Add to Plugin Marketplace

### ISSUE THREAD

**Issue by hanzei** [streamer45/mattermost-plugin-voice#12]
Would you be interested on submitting your plugin to the Plugin Marketplace? You can find the process [here](https://developers.mattermost.com/extend/plugins/community-plugin-marketplace/).

**Comment by SrMouraSilva** [comment]
Hey, the https://mattermost.com/marketplace/voice-messaging/ plugin is for this project, right?

**Comment by streamer45** [comment]
> Hey, the https://mattermost.com/marketplace/voice-messaging/ plugin is for this project, right?

Yes that is it. It's just not been added to the official marketplace yet (the one you can access in the app itself).
